### PR TITLE
Moderator Fixes / Requests

### PR DESCRIPTION
**Pull request by endrawes0**
Comparisons against moderators are now lower case. Added handling for results without gamelinks. Moderators can still post results without a link.

### CODE REVIEW

**Comment by endrawes0** [comment]
fixes #176

File: src/chesster.js [hunk 1]
```diff
@@ -1095,37 +1095,41 @@ function(bot, message) {
                     result);
             }else if(!_.isNil(pairing)){
                 //update the pairing with the result bc there was no link found
```
Comment (lakinwecker) [inline src/chesster.js]: This comment deserves to be updated

File: src/chesster.js [hunk 2]
```diff
@@ -1064,7 +1064,7 @@ function(bot, message) {
         if(
             !_.isEqual(result.white.id, message.user) &&
             !_.isEqual(result.black.id, message.user) &&
-            !message.player.isModerator()
+            !message.player.isModerator(message)
```
Comment (lakinwecker) [inline src/chesster.js]: I don't understand why message was required here. But I also haven't looked at where this is defined.

File: src/chesster.js (hunk 1 above)
Comment (endrawes0) [inline src/chesster.js]: good call.

File: src/chesster.js (hunk 2 above)
Comment (endrawes0) [inline src/chesster.js]: it compares it to the league associated with the message.

a little odd - just a helper on the player. Im not sure why or when we added it. but there it is.

**Pull request merged**